docs/components/modeler/element-templates/element-template-with-dependencies.md

```diff
@@ -8,11 +8,10 @@ When creating element templates, you may want to link to a particular resource l
 
 Element templates can depend on:
 
-- Resources:
-  - [Camunda forms](/components/modeler/forms/camunda-forms-reference.md): used in user tasks.
-  - [RPA scripts](/components/rpa/overview.md): Used in service tasks.
-  - [BPMN tasks](/components/modeler/bpmn/bpmn.md): used in call activities.
-  - [DMN decisions](/components/modeler/dmn/dmn.md): used in business rule tasks.
+- [Camunda forms](/components/modeler/forms/camunda-forms-reference.md): used in user tasks.
+- [RPA scripts](/components/rpa/overview.md): Used in service tasks.
+- [BPMN tasks](/components/modeler/bpmn/bpmn.md): used in call activities.
```

Comment:
🔧 Technically, the task is not use  **in** a call activity. It's use in the called process.  The called process is referenced by the call activity

Reply:
I think there is a misunderstanding. 
The CallActivity BPMN Element defines the process ID of the called process. 
The called process, itself a bpmn diagram, which could also have any number of dependencies, is the actual dependency. 

So it would rather be [BPMN Process]: used in a call activity

Reply:
I used the provided description 😄